### DISCUSSION TITLE
제페토 용량은 씬을 빌드하면 나오는 용량으로 확인하면되나요?

### DISCUSSION THREAD

**Discussion by yoon-hw22** [naverz/zepeto-studio-kor#1059] [5. Q&A]
제페토 용량은 씬을 빌드하면 나오는 용량으로 확인하면되나요?

**Comment by YSHFM** [comment]
유니티 프로젝트 내 상단 publish버튼을 누르면 프로젝트 내 build폴더 내에서 .zepetopackage파일이 생성됩니다.

 해당 파일 용량이 업로드 시 체크 되는 용량입니다.
